synapse/tools/genpkg.py

```diff
@@ -144,21 +147,23 @@ async def main(argv, outp=s_output.stdout):
     pars.add_argument('--no-docs', default=False, action='store_true',
                       help='Do not require docs to be present and replace any doc content with empty strings.')
     pars.add_argument('pkgfile', metavar='<pkgfile>',
-                      help='Path to a storm package prototype yml file, or a completed package JSON file.')
+                      help='Path to a storm package prototype yml file, or a completed package JSON/YAML file.')
```

Comment:
We should probably change the other reference from yml to YAML here so we can be consistent within a sentence :D

Reply:
fixed